src/Razor/src/Microsoft.VisualStudio.RazorExtension/Microsoft.VisualStudio.RazorExtension.csproj

```diff
@@ -249,6 +249,7 @@
     <!-- We need to directly reference the O# language server here because we mark it as PrivateAssets="All" in our language server itself
     We don't actual use it, but for now WebTools depends on it. -->
     <PackageReference Include="OmniSharp.Extensions.LanguageServer" Version="$(OmniSharpExtensionsLanguageServerPackageVersion)" />
+    <PackageReference Include="System.Reactive" Version="$(SystemReactivePackageVersion)" />
```

Comment:
The contents of the diff has changed since it was last opened.
First [commit](https://github.com/dotnet/razor/commit/b3626919175eda40df53ff2e7a0a1ea6b7099e6d) was removing System.Reactive. Updating title.